src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -115,13 +128,22 @@ const CoverImageEditModal = ({
           </div>
 
           {hasImage ? (
-            <div className="flex-1 flex m-8 rounded-lg items-center justify-center">
-              <img
-                src={imgSrc}
-                alt={facility.name}
-                className="w-full h-full object-cover"
-              />
-            </div>
+            <>
+              <div className="flex-1 flex m-8 rounded-lg items-center justify-center">
+                <img
+                  src={imgSrc}
+                  alt={facility.name}
+                  className="w-full h-full object-cover"
+                />
+              </div>
+              <p className="text-gray-700 font-medium text-center">
+                Max size for image uploaded should be 1mb.
+                <br />
+                Allowed formats are jpg,png,jpeg.
+                <br />
+                Recommended aspect ratio for facility cover photo is 1:1.
+              </p>
+            </>
```

Comment:
this part seems to be repeating, why not refactor it

Reply:
yeah refactored